examples/with-apollo-and-redux/lib/initClient.js

```diff
@@ -1,8 +1,6 @@
 import { ApolloClient, createNetworkInterface } from 'react-apollo'
 
-let apolloClient = null
-
-function _initClient (headers, initialState) {
+export const initClient = (headers, initialState = {}) => {
```

Comment:
I agree with you. But do you think that shouldn't also apply to the server?
So we have:
```js
export const initClient = (headers, initialState = {}) => {
  if (!apolloClient) {
    apolloClient = _initClient(headers, initialState);
  }
  return apolloClient;
}
```
P.S: The server, just as the client - needs one of those on each page/reloads